src/main/java/games/strategy/util/Md5Crypt.java

```diff
@@ -21,7 +21,7 @@
 public final class Md5Crypt {
   public static final String MAGIC = "$1$";
   private static final Pattern ENCRYPTED_PASSWORD_PATTERN =
-      Pattern.compile("^" + MAGIC.replace("$", "\\$") + "([\\.\\/a-zA-Z0-9]{1,8})\\$[\\.\\/a-zA-Z0-9]{22}$");
+      Pattern.compile("^" + MAGIC.replace("$", "\\$") + "([\\.\\/a-zA-Z0-9]{1,8})\\$([\\.\\/a-zA-Z0-9]{22})$");
```

Comment:
What's the slight change in the pattern for?

Reply:
I added a capture group around the hash portion so I can now reference it as group 2 in `getHash()`.  There was previously only a capture group around the salt for use in `getSalt()`.